config/software/openssl.rb

```diff
@@ -69,22 +77,18 @@
 
   configure_args = [
     "--prefix=#{install_dir}/embedded",
-    "--with-zlib-lib=#{install_dir}/embedded/lib",
-    "--with-zlib-include=#{install_dir}/embedded/include",
+    "no-comp",
     "no-idea",
     "no-mdc2",
     "no-rc5",
+    "no-ssl2",
+    "no-ssl3",
```

Comment:
Do we have to kill ssl3? That will mean Metasploit will no longer be able to connect to an SSLv3-only host, which, for instance, RDP can be.

Reply:
You are right, this came from upstream, I think we actually need to switch out for `enable-ssl3`.  Updating...